components/cdc/src/endpoint.rs

```diff
@@ -685,7 +686,7 @@ impl Initializer {
         let current = TimeStamp::max();
         let mut scanner = ScannerBuilder::new(snap, current, false)
             .range(None, None)
-            .build_delta_scanner(self.checkpoint_ts)
+            .build_delta_scanner(self.checkpoint_ts, ExtraOp::ReadOldValue)
```

Comment:
Looks like it reads old value by default, is it expected?

Reply:
My fault.😢